MSAL/src/MSALErrorConverter.m

```diff
@@ -201,6 +212,7 @@ + (NSError *)errorWithDomain:(NSString *)domain
         msalUserInfo[mappedKey] = userInfo[key];
     }
 
+    if (!msalUserInfo[MSIDCorrelationIdKey] && correlationId) msalUserInfo[MSIDCorrelationIdKey] = correlationId;
```

Comment:
should this be MSALCorrelationIDKey and not MSIDCorrelationIdKey?

Reply:
Yes, I messed with those two keys